jwst/datamodels/schemas/referencefile.schema.yaml

```diff
@@ -1,32 +1,203 @@
-allOf:
-- $ref: core.schema.yaml
-- type: object
-  properties:
-    meta:
-      type: object
-      properties:
-        reffile:
-          title: Information about the reference file
-          type: object
-          properties:
-            type:
-              title: Reference file type
-              type: string
-              fits_keyword: REFTYPE
-            pedigree:
-              title: The pedigree of the reference file
-              type: string
-              fits_keyword: PEDIGREE
-            description:
-              title: Description of the reference file
-              type: string
-              fits_keyword: DESCRIP
-            author:
-              title: Author of the reference file
-              type: string
-              fits_keyword: AUTHOR
-            useafter:
-              title: Use after date of the reference file
-              type: string
-              fits_keyword: USEAFTER
+type: object
+properties:
+  meta:
+    type: object
+    properties:
+      date:
+        allOf:
+        title: Date this file was created (UTC)
+        type: string
+        fits_keyword: DATE
+      origin:
+        title: Organization responsible for creating file
+        type: string
+        fits_keyword: ORIGIN
+      time_sys:
+        title: principal time system for time-related keywords
+        type: string
+        fits_keyword: TIMESYS
+      filename:
+        title: Name of the file
+        type: string
+        fits_keyword: FILENAME
+      filetype:
+        title: Type of data in the file
+        type: string
+        fits_keyword: FILETYPE
+      data_processing_software_version:
+        title: Data processing software version number
+        type: string
+        fits_keyword: SDP_VER
+      model_type:
+        title: Type of data model
+        type: string
+        fits_keyword: DATAMODL
+      telescope:
+        title: Telescope used to acquire the data
+        type: string
+        default: JWST
+        fits_keyword: TELESCOP
+      reffile:
+        title: Information about the reference file
+        type: object
+        properties:
+          type:
+            title: Reference file type
+            type: string
+            fits_keyword: REFTYPE
+          pedigree:
+            title: The pedigree of the reference file
+            type: string
+            fits_keyword: PEDIGREE
+          description:
+            title: Description of the reference file
+            type: string
+            fits_keyword: DESCRIP
+          author:
+            title: Author of the reference file
+            type: string
+            fits_keyword: AUTHOR
+          useafter:
+            title: Use after date of the reference file
+            type: string
+            fits_keyword: USEAFTER
+        required: [author, pedigree, description, useafter]
```

Comment:
The CRDS certifier already checks these so this is redundant and probably too strict.  While the file creators may indeed forget them (unless there are warnings),  these days CRDS won't allow them to be missing when files are submitted.   This was not always the case because historically many files came in "bad" with respect to trivia and I did not make it an issue because we could barely get essential matching parameters correct.  The only thing operationally critical is USEAFTER, and even that was not critical when we started out and even now is only critical when references are submitted and added to the rmaps.   It's an OK idea to issue warnings for these,  but I suspect failing them is bad if it makes "working" references start failing at runtime.